devenv/zabbix70/bootstrap/Dockerfile

```diff
@@ -1,12 +1,12 @@
-FROM python:2.7
+FROM python:3.12-slim-bullseye
 
 ENV ZBX_API_URL=http://zabbix-web:8080
 ENV ZBX_API_USER="Admin"
 ENV ZBX_API_PASSWORD="zabbix"
-ENV ZBX_CONFIG="zbx_export_hosts.xml"
+ENV ZBX_CONFIG="zbx_export_hosts.json"
 ENV ZBX_BOOTSTRAP_SCRIPT="bootstrap_config.py"
 
-RUN pip install pyzabbix
+RUN pip install zabbix_utils
```

Comment:
I changed this to the official zabbix python API because the pyzabbix was failing.